e2e/erasure_coded.go

```diff
@@ -0,0 +1,79 @@
+/*
+Copyright 2021 The Ceph-CSI Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package e2e
+
+import (
+	"fmt"
+
+	"k8s.io/kubernetes/test/e2e/framework"
+)
+
+const (
+	// number of data chunks.
+	k = 2
+	// number of coding chunks.
+	m = 1
+	// number of placement groups.
+	pgCount                 = 12
+	crushFailureDomain      = "osd"
+	erasureCodedPoolName    = "ec-pool"
+	erasureCodedProfileName = "ec-profile"
+	appName                 = "myapp"
+)
+
+// createErasureCodedPool creates an erasure coded pool.
+func createRbdErasureCodedPool(f *framework.Framework, ecname string) error {
```

Comment:
@Yuggupta27 I may be off here, but would like to ask anyway, dont rook create EC pools if CRs are adjusted?

Reply:
Yes, rook can create ec pools with a custom resource. Should we use them in ceph-csi for our e2e?

Reply:
Better figure out how to create the pool? It can serve as documentation as well...

Reply:
@Yuggupta27 looking into the cluster CR provided in the rook example, there is an option . If thats the case, its also not a bad idea to just consume it in our e2e I believe.

Reply:
Oh.. You actually made use of in the last refresh , cool :+1: , it reduced a good amount of code here !!! Thanks

Reply:
Yes @humblec , it minimizes a lot of unnecessary code :)

Reply:
Thats really good ! thanks.